src/convnets/mobilenet.jl

```diff
@@ -36,7 +36,7 @@ function mobilenetv1(width_mult, config;
         for _ in 1:nrepeats
             layer = dw ?
                     depthwise_sep_conv_bn((3, 3), inchannels, outch, activation;
-                                          stride = stride, pad = 1) :
+                                          stride = stride, pad = 1, bias = false) :
```

Comment:
Was this change intended? Is this a bug you caught?

Reply:
Yep, this is intended. MobileNetv1 still needs to be fixed a little bit, but I will leave that for a separate PR